_submissions/round-12/1/2015-03-05-openmp-outline.md

```diff
@@ -0,0 +1,15 @@
+---
+date: 2015-03-05
+round: Round 12
+title: OpenMP Topics/Outline
+author: John Pormann
+permalink: /2015/02/
```

Comment:
Needs to be the full path to the post, e.g., /2015/02/openmp-concept-map/ (with a trailing slash).